CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **capital-framework:** [PATCH] Standardized all the usage docs for v4
```

Comment:
Should this be `capital-framework` or `all components`? We should review the release script to be sure. /cc @contolini in case he has a quick answer.